README.md

```diff
@@ -74,21 +72,29 @@ class MyComponent extends React.Component<MyComponentProps, MyComponentState> {
 
 ## Usage
 
-### CLI
+### Install and configure
 
-```
-npm install -g react-js-to-ts
-```
+This repo currently needs to be cloned and built directly[\*](#original-project)
```

Comment:
Oh that's rough. We can't npm install from the github url and then `npx react-js-to-ts filename.js`?

Reply:
We can't currently install directly from github because it requires a build step. We could get around this by pushing the built files up to github or we could also publish our own version of this package to npm. I was just getting some questions on how to use this repo, so I figured I'd document how to use it currently and we can come back later and change how it is managed later once we decide on what to do.